modules/wechat_qrcode/src/detector/align.hpp

```diff
@@ -0,0 +1,42 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Tencent is pleased to support the open source community by making WeChat QRCode available.
+// Copyright (C) 2020 THL A29 Limited, a Tencent company. All rights reserved.
+
+#ifndef __DETECTOR_ALIGN_HPP_
+#define __DETECTOR_ALIGN_HPP_
+
+#include <stdio.h>
+#include <fstream>
+#include "../precomp.hpp"
```

Comment:
> #include "../precomp.hpp"

Relative path is valid, but `precomp.hpp` should not be used in .hpp files (the first include of `.cpp` files).